api/Window.json

```diff
@@ -221,31 +221,83 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": false
             },
             "ie": {
-              "version_added": null
+              "version_added": false
             },
             "opera": {
-              "version_added": null
+              "version_added": "32"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "32"
             },
             "safari": {
-              "version_added": null
+              "version_added": false
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": false
             },
             "samsunginternet_android": {
               "version_added": null
             },
             "webview_android": {
+              "version_added": "45"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "ondevicelight": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/ondevicelight",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "22",
+              "notes": "Works on Mac OSX. Support for Windows 7 <a href='https://bugzil.la/754199'in progress</a>."
```

Comment:
This seems to be out of date: looking at https://bugzilla.mozilla.org/show_bug.cgi?id=754199 and the related https://bugzilla.mozilla.org/show_bug.cgi?id=1359076, it looks like this might be disabled in Firefox now?